offline_camera_calibration:
  ros__parameters:

    minimum_valid_images: 21

    chessboard_size: [9, 7]
    square_size: 30.0 # mm

    image_size: [1280, 960] # (width, height)
    
    images_path: /home/marco/xarm_devel/src/dependencies/ROS2-HAL-Allied-Vision-Alvium1800U/allied_vision_camera/calibration_images/
    calibration_path: /home/marco/xarm_devel/src/dependencies/ROS2-HAL-Allied-Vision-Alvium1800U/allied_vision_camera/calibration/

    calibration_file: calib_test1.xml